### PR TITLE
fix(parse5adapter): Update parse5 to latest & fix functionality

### PR DESCRIPTION
**Pull request by MarkPieszak**
Fix treeAdapter
Add invoke, query, querySelector, querySelectorAll, userAgent functionality

### CODE REVIEW

File: modules/platform-node/parse5-adapter.ts [hunk 1]
```diff
@@ -8,42 +8,56 @@ const parse5: any = require('parse5');
...
 export class Parse5DomAdapter extends DomAdapter {
+
   static makeCurrent() {
-    parser = parse5;
-    serializer = parse5.serialize;
     treeAdapter = parse5.treeAdapters.htmlparser2;
     setRootDomAdapter(new Parse5DomAdapter());
   }
 
+  // Previously we had:
```
Comment (gdi2290) [inline modules/platform-node/parse5-adapter.ts]: can you remove these comments

**Comment by gdi2290** [comment]
@googlebot you crazy

File: modules/platform-node/parse5-adapter.ts (hunk 1 above)
Comment (MarkPieszak) [inline modules/platform-node/parse5-adapter.ts]: 🐗  🐎 done

**Comment by gdi2290** [comment]
I verify google bot

**Comment by MarkPieszak** [comment]
i approve @googlebot

**Comment by gdi2290** [comment]
google bot I approve

**Comment by gdi2290** [comment]
@googlebot I approve this PR

**Comment by MarkPieszak** [comment]
@googlebot engage. yes

**Comment by gdi2290** [comment]
@googlebot I am okay with these commits

**Comment by gdi2290** [comment]
@googlebot are you related to google assistant?

**Comment by MarkPieszak** [comment]
@googlebot i authored these commits. accept.

**Comment by gdi2290** [comment]
@googlebot i authored these commits. accept.

**Comment by gdi2290** [comment]
@googlebot i accept.

**Comment by gdi2290** [comment]
confirm

**Comment by GoogleAssistant** [comment]
Hi, how can I help?

**Pull request merged**